### PR TITLE
Handlers migration guide

### PR DESCRIPTION
**Pull request by waymobetta**
## Description

A guide is written to detail steps to be taken by a bridge admin in order to configure a new handler (custom erc20/721/generic) instead of the using one of the default handlers.

**Includes:**
- [x] overwrite existing resource registration; code: [adminSetResource (SOL)](https://github.com/ChainSafe/chainbridge-solidity/blob/master/contracts/Bridge.sol#L221)
- [x] migration (moneys)
- [x] gotchas?

**Questions:**
1. should this assume that the new custom handler is already deployed?
2. ~should bridge be paused during this migration?~
3. is token amount in [withdraw](https://github.com/ChainSafe/chainbridge-core/blob/main/chains/evm/cli/admin/withdraw.go#L16) command in WEI?

## Related Issue Or Context

Closes: [#8](https://github.com/ChainSafe/chainbridge-project-backlog/issues/8)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [x] I have updated the documentation locally and in chainbridge-docs.
- [ ] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

**Comment by waymobetta** [comment]
High level review is ready, but I'd also like to perform all of these steps manually after `withdraw` command is implemented following [Solidity PR 416](https://github.com/ChainSafe/chainbridge-solidity/pull/416) to ensure that nothing is being missed.

Also I would like to update the guide to use real values for the code examples, such as correctly formatted contract/wallet addresses and real amount in WEI.

File: docs/guides/handlers-migration-guide.md [hunk 1]
```diff
@@ -25,21 +21,39 @@ As a result, a dev may wish to implement their own custom handlers outside of th
...
+You can also utilize the CLI to query the balance of either: an [Externally Owned Account (EOA)](https://ethdocs.org/en/latest/contracts-and-transactions/account-types-gas-and-transactions.html) or a [Contract Account](https://ethdocs.org/en/latest/contracts-and-transactions/account-types-gas-and-transactions.html#contract-accounts). 
 
-2. Withdraw total token balance from HandlerA (old) into HandlerB (new)
+A demo example on how to do this with more context can be found [here](../transfer-and-balances.md#query-balances).
 
-Once you have the total token balance that exists on HandlerA (old), utilize the admin CLI command `withdraw` in order to transfer out the tokens that exist on HandlerA (old), effectively draining the contract of its entire token balance and setting the output of this action to be the newly deployed, custom handler, HandlerB (new).
+[Docs: ERC20 Balance](https://github.com/ChainSafe/chainbridge-core/blob/main/README.md#balance)
+
+3. Withdraw total token balance from HandlerA (old) into HandlerB (new)
```
Comment (P1sar) [inline docs/guides/handlers-migration-guide.md:54]: Maybe it worth mentioning that withdrawal is only actual in terms that Handler was Burnable?

File: docs/guides/handlers-migration-guide.md [hunk 2]
```diff
@@ -25,21 +21,39 @@ As a result, a dev may wish to implement their own custom handlers outside of th
...
+5. Unpausing the bridge contract
 ### Steps
-1. Identify total token balance stored within HandlerA (old)
+1. Pause the bridge contract
+
+In this first step, we will invoke the admin `pause` command which will freeze the bridge. During this time, both deposits and proposals will be disallowed. 
+
+This is an important step to remember to take as a bridge administrator so as to prevent your bridge users from accidentally losing tokens during the migration of your handlers.
+
+[Docs: Admin Pause](https://github.com/ChainSafe/chainbridge-core/blob/main/README.md#pause)
```
Comment (P1sar) [inline docs/guides/handlers-migration-guide.md:40]: Please create issue to add pause/unpause CLI CMDs and make sure we added them on next sprint

**Review by P1sar — CHANGES_REQUESTED** [review]
Everting  else looks good for me

File: docs/guides/handlers-migration-guide.md (hunk 1 above)
Comment (waymobetta) [reply docs/guides/handlers-migration-guide.md:54]: I'm not understanding. Can you phrase differently please?

Comment (waymobetta) [reply docs/guides/handlers-migration-guide.md:54]: I added some clarity to the docs based on your feedback

File: docs/guides/handlers-migration-guide.md (hunk 2 above)
Comment (waymobetta) [reply docs/guides/handlers-migration-guide.md:40]: Cool, will create issues and add to next sprint

File: docs/guides/handlers-migration-guide.md (hunk 1 above)
Comment (P1sar) [reply docs/guides/handlers-migration-guide.md:54]: i think its ok now

**Review by P1sar — APPROVED** [review]

**Pull request merged**